### PR TITLE
[script] [dependency] support move to `dragon-realms/dr-scripts` repo

### PR DESCRIPTION
**Pull request by KatoakDR**
### Background
* Fixes https://github.com/rpherbig/dr-scripts/issues/4888

### Changes
* Add method `determine_github_status_repo` which makes a ping to `dragon-realms/dr-scripts` and `rpherbig/dr-scripts` to determine which location is currently hosting the repository. The one that returns an http `OK` response wins :)

### Post-Move Logic
* After we've migrated to `dragon-realms/dr-scripts`, then a cleanup task will be done to find & replace strings `rpherbig/dr-scripts`
* After two weeks, remove the `determine_github_status_repo` method

### CODE REVIEW

**Review by MahtraDR — APPROVED** [review]
The change looks good, I'd say this is a change to 1.5.0, as we're making some scope changes to what the script does here.

**Comment by rcuhljr** [comment]
> After two weeks, remove the determine_github_status_repo method 
Laughs at the optimism :)

**Comment by rcuhljr** [comment]
We'll need a new `@git_token` for the other org I think.

File: dependency.lic [hunk 1]
```diff
@@ -10,7 +10,7 @@ require 'ostruct'
 require 'digest/sha1'
 require 'monitor'
 
-$DEPENDENCY_VERSION = '1.4.3'
+$DEPENDENCY_VERSION = '1.4.4'
```
Comment (KatoakDR) [inline dependency.lic:13]:
```suggestion
$DEPENDENCY_VERSION = '1.5.0'
```

Resolve https://github.com/rpherbig/dr-scripts/pull/5626#pullrequestreview-894602267

**Comment by KatoakDR** [comment]
> We'll need a new `@git_token` for the other org I think.

Yes, is that something you can provide?

**Comment by rcuhljr** [comment]
@rpherbig can probably grab one, I don't remember if I'm in the new org.

**Comment by rpherbig** [comment]
> @rpherbig can probably grab one, I don't remember if I'm in the new org.

You're an owner of the new org.

That said, I don't see where to generate a token _for the org_. If it's generate at a repository level, then I suspect we can't until the repository actually exists under the new org...

**Comment by KatoakDR** [comment]
> > @rpherbig can probably grab one, I don't remember if I'm in the new org.
> 
> You're an owner of the new org.
> 
> That said, I don't see where to generate a token _for the org_. If it's generated at a repository level, then I suspect we can't until the repository actually exists under the new org...

Tokens are created by a user at https://github.com/settings/tokens

I don't know who created the token we use today for `rpherhig/dr-scripts`, but I suspect someone with long-term access to `dragon-realms` organization should generate a read only token that doesn't expire.

**Comment by rpherbig** [comment]
Thank you @KatoakDR ! Token DM'd to you for inclusion in the PR.

**Comment by Kiriawen** [comment]
@KatoakDR @MahtraDR 

Is this still in progress or ready for review?  If you don't intend to continue with it please close it out.

Thanks!

**Comment by MahtraDR** [comment]
Closing this as a topic. We can revisit whenif it's relevant again.

**Pull request closed without merging**